website/docs/r/snat.html.markdown

```diff
@@ -11,11 +11,15 @@ description: |-
 Provides a vCloud Director SNAT resource. This can be used to create, modify,
 and delete source NATs to allow vApps to send external traffic.
 
+!> **Warning:** When advanced edge gateway is used and the rule is updated using UI, then Id mapping will be lost and terraform won't find the rule anymore and remove it from state.
```

Comment:
This catches attention nicely. 
<img width="1101" alt="image" src="https://user-images.githubusercontent.com/15804230/61533933-e34de680-aa36-11e9-9ca5-667769cf577a.png">

Reply:
nice